src/FileSystemUtilities.js

```diff
@@ -51,7 +51,10 @@ export default class FileSystemUtilities {
 
   @logger.logifyAsync()
   static rimraf(filePath, callback) {
-    rimraf(filePath, callback);
+    ChildProcessUtilities.exec("npm bin", {cwd: __dirname}, (err, npmBinPath) => {
```

Comment:
I had that thought too, and then decided that this was probably roughly equivalent to a shell script that did something like `$(npm bin)/rimraf /some/dir/node_modules`, so I didn't bother fixing it. It seems like the cost of running `rimraf` (node-based filesystem walking and unlinking) would dwarf the `npm bin` exec for most projects anyway?

If we want to cache it, I guess we'd do that in a module-level variable in FileSystemUtilities? Wouldn't be hard to add.